packages/cozy-mespapiers-lib/src/components/ModelSteps/widgets/InputTextAdapter.jsx

```diff
@@ -156,6 +156,7 @@ const InputTextAdapter = ({
           type="text"
           variant="outlined"
           label={inputLabel ? t(inputLabel) : ''}
+          id={inputLabel ? t(inputLabel) : ''}
```

Comment:
can't be done by cozy-ui?

Reply:
And why do you translate the id? 

Also, ids are supposed to be unique?

Reply:
Oui en effet, j'allais retoucher ça ce matin. C'est une erreur de ma part, un ID unique ajouté par défaut côté cozy-ui est bien plus pertinent 👍

Reply:
cf:  https://github.com/cozy/cozy-ui/pull/2401